### PR TITLE
[AV-86503] Updated the changelog

### PR DESCRIPTION
**Pull request by aniket-Kumar-c**
## Jira

* AV-86503

## Description
Update the changelog for the v1.3.0 release

Please include a summary of the fix/feature/change, including any relevant motivation and context.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [x] Documentation fix/enhancement

### How was this change tested and do you have evidence? _**(REQUIRED: Select at least 1)**_

- [ ] Manually tested
- [ ] Unit tested
- [ ] Acceptance tested
- [x] Unable to test / will not test (Please provide comments in section below)

### Testing

<details open>
  <summary>Testing</summary>
  
</details>

## Required Checklist:

- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if required)
- [x] I have run make fmt and formatted my code
- [x] I have made sure that no schema field is marked with both requiresReplace and computed
## Further comments

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
 
+## [v1.3.0](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/v1.3.0) (2024-09-11)
+
+[Full Changelog](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/compare/v1.2.1...v1.3.0)
+
+**Implemented enhancements:**
+- \[AV-85326\] Adding the connection string in the GetCluster operation [\#219](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/219) ([PaulomeeCb](https://github.com/PaulomeeCb))
+- \[AV-76499\] System events and activity logs [\#206](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/206) ([aniket-Kumar-c](https://github.com/aniket-Kumar-c))
+- \[AV-78888\] Support for VPC peering has been added for AWS and GCP [\#205](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/205) ([PaulomeeCb](https://github.com/PaulomeeCb))
+- \[AV-76500\] Implement private endpoint service [\#202](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/202) ([l0n3star](https://github.com/l0n3star))
```
Comment (rajagp) [inline CHANGELOG.md:11]: Include supported CSPs

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## [v1.3.0](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/v1.3.0) (2024-09-11)
+
+[Full Changelog](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/compare/v1.2.1...v1.3.0)
+
+**Implemented enhancements:**
+- \[AV-85326\] Adding the connection string in the GetCluster operation [\#219](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/219) ([PaulomeeCb](https://github.com/PaulomeeCb))
```
Comment (Talina06) [inline CHANGELOG.md:8]:
```suggestion
- \[AV-85326\] Display the cluster connection string for the cluster resource [\#219](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/219) ([PaulomeeCb](https://github.com/PaulomeeCb))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+- \[AV-76499\] System events and activity logs [\#206](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/206) ([aniket-Kumar-c](https://github.com/aniket-Kumar-c))
```
Comment (Talina06) [inline CHANGELOG.md:9]:
```suggestion
- \[AV-76499\] Support Capella System Events and Activity Logs [\#206](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/206) ([aniket-Kumar-c](https://github.com/aniket-Kumar-c))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+- \[AV-78888\] Support for VPC peering has been added for AWS and GCP [\#205](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/205) ([PaulomeeCb](https://github.com/PaulomeeCb))
```
Comment (Talina06) [inline CHANGELOG.md:10]:
```suggestion
- \[AV-78888\] Add Support for VPC peering for AWS and GCP [\#205](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/205) ([PaulomeeCb](https://github.com/PaulomeeCb))
```

File: CHANGELOG.md (hunk 1 above)
Comment (Talina06) [inline CHANGELOG.md:11]:
```suggestion
- \[AV-76500\] Support for Private Endpoints [\#202](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/202) ([l0n3star](https://github.com/l0n3star))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,32 @@
...
+- \[AV-76500\] Implement private endpoint service [\#202](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/202) ([l0n3star](https://github.com/l0n3star))
+- \[AV-79411\] Custom user agent in HTTP header [\#200](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/200) ([l0n3star](https://github.com/l0n3star))
```
Comment (Talina06) [inline CHANGELOG.md:12]:
```suggestion
- \[AV-79411\] Add a Custom User-Agent HTTP header for Terraform Provider Client [\#200](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/200) ([l0n3star](https://github.com/l0n3star))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,32 @@
...
+
+**Fixed bugs:**
+- \[AV-85326\] Fixed Private Endpoint Service example [\#218](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/218) ([l0n3star](https://github.com/l0n3star))
```
Comment (Talina06) [inline CHANGELOG.md:15]:
```suggestion
- \[AV-85326\] Fix Private Endpoints examples [\#218](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/218) ([l0n3star](https://github.com/l0n3star))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,32 @@
...
+- \[AV-84160\] Make cluster timezone optional [\#215](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/215) ([Lagher0](https://github.com/Lagher0))
+- \[AV-83067\] Remove server version check for acceptance tests [\#210](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/210) ([l0n3star](https://github.com/l0n3star))
+- \[AV-83066\] Resolving getting started folder bugs [\#209](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/209) ([PaulomeeCb](https://github.com/PaulomeeCb))
```
Comment (Talina06) [inline CHANGELOG.md:18]:
```suggestion
- \[AV-83066\] Resolve getting-started folder bugs [\#209](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/209) ([PaulomeeCb](https://github.com/PaulomeeCb))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,32 @@
...
+- \[AV-82517\] Fixing acceptance tests [#207](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/207) ([PaulomeeCb](https://github.com/PaulomeeCb))
```
Comment (Talina06) [inline CHANGELOG.md:19]:
```suggestion
- \[AV-82517\] Fix acceptance tests [#207](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/207) ([PaulomeeCb](https://github.com/PaulomeeCb))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,32 @@
...
+- \[AV-79969\] Use noop when resource is destroyed for audit logs [\#203](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/203) ([l0n3star](https://github.com/l0n3star))
+
+**Documentation Enhancement:**
+- \[AV-83061\] Updating contributing.md with windows steps [#208](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/208) ([Talina06](https://github.com/Talina06))
```
Comment (Talina06) [inline CHANGELOG.md:23]:
```suggestion
- \[AV-83061\] Update contributing.md with windows steps [#208](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/208) ([Talina06](https://github.com/Talina06))
```

**Review by Talina06 — COMMENTED** [review]
changelog comments should be in present tense, designed for human readability. @aniket-Kumar-c I recommend updating the PR titles to match the suggestions, and then to regenerate the changelog.

**Review by nishant-tripathy — APPROVED** [review]
LGTM

**Review by Talina06 — APPROVED** [review]
lgtm

**Pull request merged**